### DISCUSSION TITLE
BT-756 is false in XML but shows as true on PDF

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#355] [Q&A] [answered]
TerminatedIndicator (BT-756) is set false in eForm 29 XML:
```
<cac:TenderingProcess>
        <cbc:Description languageID="EST">Piiratud hankemenetlus</cbc:Description>
        <cbc:ProcedureCode listName="procurement-procedure-type">restricted</cbc:ProcedureCode>
        <cbc:TerminatedIndicator>false</cbc:TerminatedIndicator>
        <cac:ProcessJustification>
            <cbc:Description>21d2fdd9-c707-4571-98e7-648516ca41f1</cbc:Description>
        </cac:ProcessJustification>
</cac:TenderingProcess>
```
Notice PDF still shows "Call for competion is terminated":

2.1.4 Üldine teave
**Hanke väljakuulutamine on lõpetatud.**
Lisateave: SEOTUD HANKE LISATEAVE
Õiguslik alus
Direktiiv 2014/24/EL 

Notice ID
<cbc:ID schemeName="notice-id">483e2966-a786-4cb6-818e-cddaaec6c9c9</cbc:ID>

**Comment by pdonohoe** [comment, marked as answer]
Thank you for reporting this issue. This is a bug in the EFX; it will be fixed in SDK 1.7.
